eng/Versions.props

```
@@ -140,7 +140,8 @@
     <MoqPackageVersion>4.18.4</MoqPackageVersion>
     <XunitCombinatorialVersion>1.3.2</XunitCombinatorialVersion>
     <MicrosoftDotNetInstallerWindowsSecurityTestDataPackageVersion>8.0.0-beta.23607.1</MicrosoftDotNetInstallerWindowsSecurityTestDataPackageVersion>
-    <BenchmarkDotNetPackageVersion>0.14.0</BenchmarkDotNetPackageVersion>
+    <BenchmarkDotNetPackageVersion>0.15.6</BenchmarkDotNetPackageVersion>
+    <BenchmarkDotNetDiagnosticsWindowsPackageVersion>0.14.0</BenchmarkDotNetDiagnosticsWindowsPackageVersion>
```

```suggestion
    <BenchmarkDotNetDiagnosticsWindowsPackageVersion>0.15.6</BenchmarkDotNetDiagnosticsWindowsPackageVersion>
```

If these packages version together we may want to use a single property to ensure they don't drift.